rpcs3/Emu/Cell/Modules/cellOskDialog.cpp

```diff
@@ -85,7 +115,7 @@ error_code cellOskDialogLoadAsync(u32 container, vm::ptr<CellOskDialogParam> dia
 
 	bool result = false;
 
-	osk->on_close = [maxLength, wptr = std::weak_ptr<OskDialogBase>(osk)](s32 status)
+	osk->on_osk_close = [maxLength, wptr = std::weak_ptr<OskDialogBase>(osk)](s32 status)
```

Comment:
the native osk has a chance to commit sudoku in the while loop containing a yield inside the osk_confirm_callback section